### PR TITLE
Add bot config setting to disable tweet posting

### PR DESCRIPTION
**Pull request by ALazyMeme**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable
- [x] I have tested all changes

This basically controls write configuration for the twitter module. It omits the need for multiple API keys.

### CODE REVIEW

**Comment by ALazyMeme** [comment]
The whispers didn't work on my bot due to the phone number requirement. Based on the error message in consol, I'd say it would've sent.

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` w
 - Bugfix: Fix bot not correctly tracking online/offline state of stream due to TwitchGame not being deserialized properly. (#2243)
 - Bugfix: Exclude deleted accounts from Twitch VIP & Moderators lists, and try to handle empty usernames better in other places. (#2319)
 - Bugfix: Fix bot not handling missing streamer token when trying to refresh moderators. (#2324)
+- Minor: Add setting to control read/write toggle on Twitter module. (#2336)
```
Comment (pajlada) [inline CHANGELOG.md]: Need to update the changelog entry to match the new direction the PR took

File: pajbot/bot.py [hunk 2]
```diff
@@ -254,6 +254,7 @@ def __init__(self, config: cfg.Config, args: argparse.Namespace) -> None:
             socket_manager=self.socket_manager, module_manager=self.module_manager, bot=self
         ).load()
         self.websocket_manager = WebSocketManager(self)
+        self.twitter_disallow_write = config["twitter"].get("disallow_write", "0")
```
Comment (pajlada) [inline pajbot/bot.py:257]: use `cfg.get_boolean` here instead

**Review by pajlada — CHANGES_REQUESTED** [review]

**Comment by ALazyMeme** [comment]
Updated behaviour seems to work just fine
```
[2023-02-19 20:14:09,705] [ERROR  ] Disallow writing set to 1 in config, will not send tweet.
[2023-02-19 20:14:09,894] [ERROR  ] Failed to send whisper, unauthorized: 401 Client Error: Unauthorized for url: https://api.twitch.tv/helix/whispers?from_user_id=196500227&to_user_id=103973901 - {"error":"Unauthorized","status":401,"message":"the sender does not have a verified phone number"}
```

**Review by pajlada — APPROVED** [review]

**Pull request merged**